Cargo.toml

```diff
@@ -48,7 +48,12 @@ anyhow = "1.0.86"
 cargo-generate = "0.23"
 
 # Better CLI user input
-dialoguer = { version = "0.11.0", default-features = false }
+dialoguer = { version = "0.11.0", default-features = false, features = [
+    "fuzzy-select",
+] }
+
+# Used to create styled table outputs.
+comfy-table = "7.1.4"
```

Comment:
The table is certainly cute! I'm not sure if it's worth pulling in a completely new dependency, though. Let me see how bearable it is to make tables using `format!()`.